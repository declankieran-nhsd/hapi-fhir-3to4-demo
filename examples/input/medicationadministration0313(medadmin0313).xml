<?xml version="1.0" encoding="UTF-8"?><MedicationAdministration xmlns="http://hl7.org/fhir"> 
  <text><status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: medadmin0313</p><p><b>status</b>: completed</p><p><b>medication</b>: Paracetamol 240mg suppository (product) <span>(Details : {SNOMED CT code &#39;322254008&#39; = &#39;Paracetamol 240mg suppository&#39;, given as &#39;Paracetamol 240mg suppository (product)&#39;})</span></p><p><b>subject</b>: <a>Donald Duck</a></p><p><b>effective</b>: 15/01/2015 10:03:00 PM --&gt; 16/01/2015 2:03:00 AM</p><h3>Performers</h3><table><tr><td>-</td><td><b>Actor</b></td></tr><tr><td>*</td><td><a>Patrick Pump</a></td></tr></table><p><b>reasonCode</b>: Emergency <span>(Details : {http://hl7.org/fhir/reason-medication-given code &#39;c&#39; = &#39;Emergency&#39;, given as &#39;Emergency&#39;})</span></p><p><b>prescription</b>: <a>MedicationRequest/medrx0324</a></p><h3>Dosages</h3><table><tr><td>-</td><td><b>Text</b></td><td><b>Site</b></td><td><b>Dose</b></td></tr><tr><td>*</td><td>Insert one suppository rectally twice daily as needed for fever to a maximim of 3 per day</td><td>Rectum structure <span>(Details : {SNOMED CT code &#39;34402009&#39; = &#39;Rectum&#39;, given as &#39;Rectum structure&#39;})</span></td><td>240 mg<span> (Details: UCUM code mg = &#39;mg&#39;)</span></td></tr></table></div></text><status value="completed"/>
  <medicationCodeableConcept>
    <coding>
      <system value="http://snomed.info/sct"/>
      <code value="322254008"/>
      <display value="Paracetamol 240mg suppository (product)"/>
    </coding>
  </medicationCodeableConcept>
 <subject>
   <reference value="Patient/pat1"/>
   <display value="Donald Duck"/> 
  </subject>
 <effectivePeriod>
  <start value="2015-01-15T22:03:00+01:00"/>
   <end value="2015-01-16T02:03:00+01:00"/>
</effectivePeriod>
  <performer>
    <actor>
      <reference value="Practitioner/f007"/>
      <display value="Patrick Pump"/>
    </actor>
  </performer>
  <reasonCode>
    <coding>
      <system value="http://hl7.org/fhir/reason-medication-given"/>
      <code value="c"/>
      <display value="Emergency"/>
    </coding>
  </reasonCode>
  <prescription>
    <reference value="MedicationRequest/medrx0324"/>
  </prescription>
  <dosage>
    <text value="Insert one suppository rectally twice daily as needed for fever to a maximim of 3 per day"/>
    <site>
      <coding>
        <system value="http://snomed.info/sct"/>
        <code value="34402009"/>
        <display value="Rectum structure"/>
      </coding>
    </site>
    <dose>
      <value value="240"/>
      <unit value="mg"/>
      <system value="http://unitsofmeasure.org"/>
      <code value="mg"/>
    </dose>
  </dosage>
</MedicationAdministration>