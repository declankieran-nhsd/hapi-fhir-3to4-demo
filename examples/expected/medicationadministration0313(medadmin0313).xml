<?xml version="1.0" encoding="UTF-8"?>

<MedicationAdministration xmlns="http://hl7.org/fhir">
  <text>
    <status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml"><p><b>Generated Narrative with Details</b></p><p><b>id</b>: medadmin0313</p><p><b>status</b>: completed</p><p><b>medication</b>: Paracetamol 240mg suppository (product) <span>(Details : {SNOMED CT code '322254008' = 'Paracetamol 240mg suppository', given as 'Paracetamol 240mg suppository (product)'})</span></p><p><b>subject</b>: <a>Donald Duck</a></p><p><b>effective</b>: 15/01/2015 10:03:00 PM --&gt; 16/01/2015 2:03:00 AM</p><h3>Performers</h3><table><tr><td>-</td><td><b>Actor</b></td></tr><tr><td>*</td><td><a>Patrick Pump</a></td></tr></table><p><b>reasonCode</b>: Emergency <span>(Details : {http://hl7.org/fhir/reason-medication-given code 'c' = 'Emergency', given as 'Emergency'})</span></p><p><b>prescription</b>: <a>MedicationRequest/medrx0324</a></p><h3>Dosages</h3><table><tr><td>-</td><td><b>Text</b></td><td><b>Site</b></td><td><b>Dose</b></td></tr><tr><td>*</td><td>Insert one suppository rectally twice daily as needed for fever to a maximim of 3 per day</td><td>Rectum structure <span>(Details : {SNOMED CT code '34402009' = 'Rectum', given as 'Rectum structure'})</span></td><td>240 mg<span> (Details: UCUM code mg = 'mg')</span></td></tr></table></div>
  </text>
  <extension url="http://hl7.org/fhir/3.0/StructureDefinition/extension-MedicationAdministration.status">
    <valueCode value="completed"/>
  </extension>
  <subject>
    <reference value="Patient/pat1"/>
    <display value="Donald Duck"/>
  </subject>
  <effectivePeriod>
    <start value="2015-01-15T22:03:00+01:00"/>
    <end value="2015-01-16T02:03:00+01:00"/>
  </effectivePeriod>
  <medicationCodeableConcept>
    <coding>
      <system value="http://snomed.info/sct"/>
      <code value="322254008"/>
      <display value="Paracetamol 240mg suppository (product)"/>
    </coding>
  </medicationCodeableConcept>
  <performer>
    <actor>
      <reference value="Practitioner/f007"/>
      <display value="Patrick Pump"/>
    </actor>
  </performer>
  <reasonCode>
    <coding>
      <system value="http://hl7.org/fhir/reason-medication-given"/>
      <code value="c"/>
      <display value="Emergency"/>
    </coding>
  </reasonCode>
  <request>
    <reference value="MedicationRequest/medrx0324"/>
  </request>
  <dosage>
    <text value="Insert one suppository rectally twice daily as needed for fever to a maximim of 3 per day"/>
    <site>
      <coding>
        <system value="http://snomed.info/sct"/>
        <code value="34402009"/>
        <display value="Rectum structure"/>
      </coding>
    </site>
    <dose>
      <value value="240"/>
      <unit value="mg"/>
      <system value="http://unitsofmeasure.org"/>
      <code value="mg"/>
    </dose>
  </dosage>
</MedicationAdministration>